### PR TITLE
Initialize comprehensive theming volumes for devstack_docker services…

### PR DESCRIPTION
**Pull request by ztraboo**
Setup comprehensive theming volumes for devstack_docker services to also include the watcher containers.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @ztraboo! I've created [OSPR-2916](https://openedx.atlassian.net/browse/OSPR-2916) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).



We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

**Comment by natabene** [comment]
@ztraboo Thank you for the contribution! When you have a chance, could you please send in a signed Contributor Agreement mentioned in the earlier comment? You only need to do this once and we can then review all your future code contributions.

**Comment by ztraboo** [comment]
@natabene I sent my signed CA document into edX legal yesterday afternoon.

From: Zachary J Trabookis <ztraboo@clemson.edu>
Sent: Tuesday, December 18, 2018 1:52 PM
To: legal@edx.org
Subject: Open edX - Contributor Agreement - Zachary Trabookis (ztraboo)

**Comment by natabene** [comment]
@ztraboo  Thank you, once our team processes it, I will post an update here.

**Comment by natabene** [comment]
@ztraboo  Thank you for joining the Open edX community! We have added your user to our records and can now review your code going forward. @edx/devops Could you give this a look?

**Comment by edx-webhook** [comment]
Thanks for the pull request, @ztraboo! I've created [OSPR-3597](https://openedx.atlassian.net/browse/OSPR-3597) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Comment by abhiyan52** [comment]
Hey, I want to change static themes files in my docker container in order to customize the theme. How can I do that without docker copy and restarting

**Comment by natabene** [comment]
> Hey, I want to change static themes files in my docker container in order to customize the theme. How can I do that without docker copy and restarting

@nedbat Could you help us out here?

**Comment by natabene** [comment]
@jmbowman Could you give a look?

File: Makefile [hunk 1]
```diff
@@ -74,7 +74,7 @@ dev.repo.reset: ## Attempts to reset the local repo checkouts to the master work
 	$(WINPTY) bash ./repo.sh reset
 
 dev.up: | check-memory ## Bring up all services with host volumes
-	bash -c 'docker-compose -f docker-compose.yml -f docker-compose-host.yml up -d'
+	bash -c 'docker-compose -f docker-compose.yml -f docker-compose-host.yml -f docker-compose-themes.yml up -d'
```
Comment (jmbowman) [inline Makefile:75]: If we add this here, it seems like it should also be added to each of the commands in this file that use `-f docker-compose-host.yml` (especially `dev.provision.run` and `dev.up.%`).  Probably in `destroy.sh` and `marketing.mk` also.

**Review by jmbowman — APPROVED** [review]
This looks ok to me; most people won't have access to the edx-themes repository, but it shouldn't hurt anything if absent (an empty directory will just get created on the host system).

**Pull request merged**